intranet/static/js/swnotifications.js

```diff
@@ -1,56 +1,55 @@
 /* global $ */
-if(window.ion.authenticated && !window.ion.gcm_optout) {
+if (window.ion.authenticated && !window.ion.gcm_optout) {
     window.addEventListener('load', function() {
-        if('serviceWorker' in navigator) {
-            navigator.serviceWorker.register('/serviceworker.js', { scope: './' }).then(subscribe);
+        if ('serviceWorker' in navigator) {
+            navigator.serviceWorker.register('/serviceworker.js', {
+                scope: './'
+            }).then(subscribe);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["subscribe" was used before it was defined](https://www.codacy.com/app/tjcsl/ion/file/2747127051/issues/source?bid=3176160&fileBranchId=3355598#l7)